content/docs/user-guide/assets/asset-processor/configuration.md

```diff
@@ -6,20 +6,56 @@ weight: 300
 toc: true
 ---
 
-You can modify the engine settings registry file for **Asset Processor** in `Registry/AssetProcessorPlatformConfig.setreg` to perform the following tasks:
+Many of the content processing rules that **Asset Processor** uses for your project are controlled by settings registry values. You can view the default values in [`Registry/AssetProcessorPlatformConfig.setreg`](https://github.com/o3de/o3de/blob/development/Registry/AssetProcessorPlatformConfig.setreg). This file is well commented, and full of examples you can use for specific configuration on your project's asset pipeline.
 
-* Add new source asset types for Asset Processor.
-* Update existing file type rules.
-* Update the ignore list.
-* Add or remove target platforms.
+To process assets for a non-host platform, you can enable additional platforms under the settings registry path `Amazon/AssetProcessor/Settings/Platforms`. Note that this is only one step in enabling a platform for your project. See the [platforms](/docs/user-guide/platforms/) section of the documentation for how to set up and deploy to additional platforms.

```

Comment:
To avoid this page looking like a wall of text, I'd recommend breaking it up a little and calling out each setting - perhaps in a level 3 header so that the font size isn't too big. For example:
```suggestion
### Platforms setting

To process assets for a non-host platform, you can enable additional platforms under the settings registry path `Amazon/AssetProcessor/Settings/Platforms`. Note that this is only one step in enabling a platform for your project. See the [platforms](/docs/user-guide/platforms/) section of the documentation for how to set up and deploy to additional platforms.
```

Reply:
I added a bunch of headers. I kind of wanted to put info in a table, but this markup language doesn't really make tables feel easy to add.

Reply:
I think it looks a lot better with the headings. I think you would have found that the text + the example code inside of the code blocks would have made the table rows look too busy and messy.